publisher/publisher.go

```diff
@@ -80,6 +81,14 @@ func (pub *publisher) GetConfigPolicy() (*cpolicy.ConfigPolicy, error) {
 	r2.Description = "KairosDB port"
 	config.Add(r2)
 
+	r3, err := cpolicy.NewBoolRule("useDynamic", true)
```

Comment:
I've updated small test with the `useDynamic `parameter. I will try to make for tests cases, but I'm really not comfortable with go...